revscoring/utilities/train_test.py

```diff
@@ -127,8 +127,10 @@ def _train_test(scorer_model, observations, test_prop):
 
     logger.info("Training model...")
     scorer_model.train(train_set)
+    scorer_model.train_set = train_set
```

Comment:
Would likely take up a lot of space to store this and the test set.  I'm not sure I understand why we'd need to store either the training or test set.